clients/scanoss/src/main/kotlin/ScanOssService.kt

```diff
@@ -34,6 +34,8 @@ import retrofit2.http.Multipart
 import retrofit2.http.POST
 import retrofit2.http.Part
 
+typealias FullScanResponse = Map<String, List<ScanResponse>>
```

Comment:
Please move the introduction of this `typealias` to a separate preceding commit, so that this commit really only touches files in the "scanner" directory.